src-shared/db-enum.js

```diff
@@ -41,9 +41,6 @@ exports.packageOptionCategory = {
   profileCodes: 'profileCodes',
   validationTimersFlags: 'validationTimersFlags',
   ui: 'ui',
-  helperCategories: 'helperCategories',
-  helperAliases: 'helperAliases',
-  resources: 'resources',
```

Comment:
Are the removals here related to your changes?

Reply:
no. it was an accident which I just fixed